src/reducers/events.js

```diff
@@ -4,15 +4,101 @@ const events_test_data = [{name: "Soccer game", location: "Empire Field, Vancouv
                           {name: "MTG Tournament", location: "Vancouver Convention Centre", date: new Date("2020-01-04"), genre: ['games', 'card games', 'tournaments', 'magic'], creator: {name: "Joan Smith"}, participants: [{name: "Daniel Boyce"}, {name: "Carol Wisnewski"}, {name: "Samuel Zubrus"}, {name: "Gary Lee"}, {name: "Andrew Gergich"}, {name: "Jennifer Quincey"}, {name: "Joel Quincey"}, {name: "Joel Harland"}, {name: "Carla Gregg"}], description: "Magic The Gathering tournament. Winner takes home $25,000 grand prize", comments: []}];
 
 
+
+
+const addParticipant = (user, event, events) => {
+        let j = -1;
+        for (let i = 0; i < events.length; i++) {
+            if (JSON.stringify(events[i]) === JSON.stringify(event)) {
```

Comment:
Do you guys want to use lodash for object equality checks instead? JSON.stringify accounts for order, so 2 objects with same key value pairs will be different if the order is different.

{"firstName": "hi", "lastName": "ho"} != {"lastName": "ho", "firstName": "hi"} with stringify.